apple/testing/default_runner/ios_test_runner.template.sh

```diff
@@ -257,11 +257,30 @@ else
   )
 fi
 
+pre_action_binary=%(pre_action_binary)s
+if [[ -x "${pre_action_binary:-}" ]]; then
+  env \
+    "$pre_action_binary"
```

Comment:
Why `env`?

Reply:
It's a habit from the fact that I daily-drive fish as my shell, and this is what you have to do, but more importantly I've found it to exhibit less ambiguous behavior in posix shells when running programs with scoped environment variables. If it would make the code more consistent, though, I'll adjust it. I'm not going to die on this hill.